tools/pipelines/templates/build-npm-package.yml

```diff
@@ -438,81 +438,32 @@ stages:
-
-            - task: npmAuthenticate@0
-              displayName: 'npm authenticate (internal feed)'
-              inputs:
-                workingFile: ${{ variables.toolAbsolutePath }}/.npmrc
-
-            - task: Bash@3
-              displayName: 'List report.json and .npmrc'
+              displayName: 'List report.json'
               inputs:
                 targetType: 'inline'
                 workingDirectory:  ${{ parameters.buildDirectory }}
                 script: |
-                  echo "Build Directory is ${{ parameters.buildDirectory }}"
-                  ls -la '${{ parameters.buildDirectory }}/artifacts/bundleAnalysis/@fluid-example/bundle-size-tests';
-                  cat '${{ variables.toolAbsolutePath }}/.npmrc';
+                  echo "Build Directory is ${{ parameters.buildDirectory }}";
+                  BUNDLE_SIZE_TESTS_DIR="${{ parameters.buildDirectory }}/artifacts/bundleAnalysis/@fluid-example/bundle-size-tests";
+                  echo "Contents of $BUNDLE_SIZE_TESTS_DIR:";
+                  ls -la '$BUNDLE_SIZE_TESTS_DIR';
 
-            - task: Bash@3
-              displayName: 'Prepare telemetry-generator for bundle size'
-              inputs:
-                targetType: 'inline'
-                workingDirectory:  ${{ variables.toolAbsolutePath }}
-                script: |
-                  ls -la '${{ variables.toolAbsolutePath }}';
-                  ls -la '${{ parameters.buildDirectory }}';
-                  npm install @ff-internal/aria-logger;
-                  npm i;
-                  npm run build:compile;
+            - template: include-telemetry-setup.yml
+              parameters:
+                devFeedUrl: $(ado-feeds-dev)
+                officeFeedUrl: $(ado-feeds-office)
 
             - task: Bash@3
               displayName: Write bundle sizes measurements to Aria/Kusto
               inputs:
                 targetType: 'inline'
-                workingDirectory: ${{ variables.toolAbsolutePath }}
+                workingDirectory: $(absolutePathToTelemetryGenerator)
                 script: |
                   echo "Writing the following performance tests results to Aria/Kusto"
                   echo "Report Size:"
                   ls -la '../../examples/utils/bundle-size-tests/bundleAnalysis/report.json';
-                  echo "Tools Path:"
-                  ls -la '../../tools/telemetry-generator/dist/handlers';
-                  node --require @ff-internal/aria-logger bin/run --handlerModule $(Build.SourcesDirectory)/tools/telemetry-generator/dist/handlers/bundleSizeHandler.js --dir '../../artifacts/bundleAnalysis/@fluid-example/bundle-size-tests';
-                  echo "Removing .npmrc"
-                  rm ../../tools/telemetry-generator/.npmrc;
+                  node --require @ff-internal/aria-logger bin/run --handlerModule dist/handlers/bundleSizeHandler.js --dir '../../artifacts/bundleAnalysis/@fluid-example/bundle-size-tests';
```

Comment:
Simplifying:
- Outputs are not as useful now (vs when initially troubleshooting).
- Removing the `.npmrc` file to appease the "extraneous modified files" check in the pipeline shouldn't be necessary (it ignores `.npmrc` files)